client/src/components/CreateForm/CreateForm.js

```diff
@@ -519,57 +519,53 @@ export default class CreateForm extends React.Component {
                 dateInput = <div/>;
         }
 
-        const referenceInputs = [];
-
-        references.forEach((reference, index) => {
-            referenceInputs.push(
-                <div className='reference-container' key={index}>
-                    <Form.Label>Reference:</Form.Label>
-                    <Form.Control
-                        type='text'
-                        name={'reference-' + index}
-                        placeholder='Reference URL'
-                        value={reference.value}
-                        onChange={(event) => this.handleReferenceChange(event)}
-                        isInvalid={!reference.isValid}
-                        className='text-control'
-                    />
-                    <Form.Control.Feedback type='invalid'>{reference.message}</Form.Control.Feedback>
-                </div>
-            );
-        });
+        const referenceInputs = references.map((reference, index) => (
+            <div className="reference-container" key={index}>
+                <Form.Label>Reference:</Form.Label>
+                <Form.Control
+                    type="text"
+                    name={'reference-' + index}
```

Comment:
Isn't this considered JSX?

Reply:
Yes but it's within curly brackets so it's JavaScript. I only replaced ones that are just strings since that's more like regular html